### ISSUE TITLE
Upstream changes

### ISSUE THREAD

**Issue by porst17** [IMAGINARY/xbanish-hide#1]
We are currently maintaining a fork of `xbanish` that enables permanent mouse pointer hiding until the process exits.

Please clean up your changes and turn them into an optional command line setting such that our changes can be merged upstream via pull-request.

**Comment by malex984** [comment]
any proposal about that _optional command line setting_?
my variant: `-a`  (for _always_ hide)...

**Comment by porst17** [comment]
I agree with `-a/--always`. But it doesn't really matter. The actual name of the option can be discussed in the pull request and can be changed in a matter of seconds once we agree with upstream on the name.

**Comment by malex984** [comment]
please check the feature branch:
https://github.com/jcs/xbanish/compare/master...IMAGINARY:feature/always_hide

**Comment by porst17** [comment]
Good to go I'd say.

**Comment by malex984** [comment]
https://github.com/jcs/xbanish/pull/35
